ports/raspberrypi/boards/sparkfun_thing_plus_rp2350/pins.c

```diff
@@ -0,0 +1,63 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2024 Scott Shawcroft for Adafruit Industries
+//
+// SPDX-License-Identifier: MIT
+
+#include "shared-bindings/board/__init__.h"
+
+static const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_GPIO0) },
```

Comment:
We prefer to group aliases of the same pin together:
```
    { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_GPIO0) },
    { MP_ROM_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_GPIO0) },
```
(the name that comes FIRST in this file is the one shown when printing the pin.

Reply:
Ah! I was unaware of that, good to know! Will get that updated, thanks!

Reply:
Ok, how's it look now?